### PR TITLE
Create chain-co.md

### PR DESCRIPTION
**Pull request by Fatman13**

### CODE REVIEW

File: open-grant-proposals/chain-co.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Grant Proposal: `A high availability solution for Filecoin Daemon`
+
+**Name of Project: Chain-co**
+
+**Proposal Category:** `core-dev`
+
+<!-- grants分类https://github.com/filecoin-project/devgrants/tree/master/open-grants -->
+
+**Proposer:** `replace with your GitHub username`
```
Comment (dtynn) [inline open-grant-proposals/chain-co.md:9]: 需要替换掉么？

File: open-grant-proposals/chain-co.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+<!-- - Start with the need or problem you are trying to solve with this project. -->
+<!-- - Describe why your solution is going to adequately solve this problem. -->
+
+At the time of writting this proposal, Filecoin network has reached ~16 EiB storage power with ~20 to ~30 PiB growth per day. Given that more than 400 nodes are having 10+ PiB of storage power with top nodes even having 100+ PiB storage power, a high availability solution for Filecoin daemon would be crucial to the stability and overall health of the network. Imagine the Filecoin daemon node for a 100 PiB storage system failed to sync height with network or experienced severe hardware failure for an extended period of time, storage power would be wiped out and causing turbulence to the network, which is not ideal at all.
+
+In order to mitigate the risk of Filecoin daemon failure and to secure a generally sane and coherent network, we propose chain coordination, chain-co for short, to coordinate a collection of redundant daemon nodes and let the storage system to use the daemon that has the most height synced. Thus, paves the way for a stronger and more resilient Filecoin network.
```
Comment (dtynn) [inline open-grant-proposals/chain-co.md]: `chain coordination` or `chain coordinator`

Comment (dtynn) [reply open-grant-proposals/chain-co.md]: most weighed?

File: open-grant-proposals/chain-co.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+## Total Budget Requested
+
+<!--Sum up the total requested budget across all milestones, and include that figure here. Also, please include a budget breakdown to specify how you are planning to spend these funds. -->
+
+## Maintenance and Upgrade Plans
+
+<!-- Specify your team's long-term plans to maintain this software and upgrade it over time. -->
+
+Chain-co will be committed to integrate the daemons of all four implementation of Filecoin to offer community a complete the high availability solution.
```
Comment (dtynn) [inline open-grant-proposals/chain-co.md]: is the `the` necessary in `a complete the high availability solution`?

File: open-grant-proposals/chain-co.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+<!-- - What are the risks that will make executing on this project difficult? -->
+
+<!-- This section should be 1-3 paragraphs long. -->
+
+## Deliverables
+
+<!-- Please describe in details what your final deliverable for this project will be. Include a specification of the project and what functionality the software will deliver when it is finished. -->
+
+Chain-co solution will be running in between a pool of redundant Fileccoin daemon of different flavors and a proxy agent.
```
Comment (dtynn) [inline open-grant-proposals/chain-co.md]: plural forms?

File: open-grant-proposals/chain-co.md (hunk 1 above)
Comment (Fatman13) [reply open-grant-proposals/chain-co.md:9]: 看看以谁的名义提交这个grant申请，就填谁。

**Comment by Fatman13** [comment]
@hunjixin 大神有空也看看？

**Pull request closed without merging**